### DISCUSSION TITLE
Sync

### DISCUSSION THREAD

**Discussion by iamLazyCode** [ImranR98/Obtainium#2000] [General]
Is there a possibility to sync the added list apps across multiple devices ?

**Comment by DwainZwerg** [comment]
The added apps are included in the Android backup (I have tried this myself). Otherwise: No and there have already been Fetaure requests for this, which were answered with the fact that this is currently out of scope.

**Reply by iamLazyCode** [reply]
Ohhh thanks alot for info

**Reply by rdscorreia74** [reply]
> this is currently out of scope

That is very sad...
I wish all apps had a backup/restore or export/import feature. I think this is the kind of feature you would put on a Pro/paid version.

**Reply by ImranR98** [reply]
We already have import/export, which is different from sync.
Also someone has said they may contribute this feature in the future - see #2013.

**Comment by s44237266-glitch** [comment]
Yes—if the app supports cloud sync (via an account, iCloud, Google Drive, or its own server), your added list apps will sync automatically across all devices logged into the same account.
Check the app’s settings for Sync, Backup, or Account options of [Redz Hub](https://redzhubscriptbloxfruit.com/). If there’s no built-in sync, you’ll need to manually export/import your lists on each device.
